### PR TITLE
Vastly simplify titanium project, add latest SDK

### PR DESCRIPTION
**Pull request by cianclarke**

### CODE REVIEW

File: Resources/app.js [hunk 1]
```diff
@@ -1,60 +1,69 @@
-/*
```
Comment (cianclarke) [inline Resources/app.js:1]: Review me - other files probably don't matter

**Comment by cianclarke** [comment]
@wei-lee another one for review :-)

**Comment by wei-lee** [comment]
Reviewed. Looks good

**Comment by cianclarke** [comment]
thanks @wei-lee

**Pull request merged**